docs/java-rest/high-level/indices/get_settings.asciidoc

```diff
@@ -0,0 +1,96 @@
+how wildcard expressions are expanded
+
+[[java-rest-high-get-settings-sync]]
+==== Synchronous Execution
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-settings-execute]
+--------------------------------------------------
+
+[[java-rest-high-get-settings-async]]
+==== Asynchronous Execution
+
+The asynchronous execution of a Get Settings request requires both the `GetSettingsRequest`
+instance and an `ActionListener` instance to be passed to the asynchronous
+method:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-settings-execute-async]
+--------------------------------------------------
+<1> The `GetSettingsRequest` to execute and the `ActionListener` to use when
+the execution completes
+
+The asynchronous method does not block and returns immediately. Once it is
+completed the `ActionListener` is called back using the `onResponse` method
+if the execution successfully completed or using the `onFailure` method if
+it failed.
+
+A typical listener for `GetSettingsResponse` looks like:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-settings-execute-listener]
+--------------------------------------------------
+<1> Called when the execution is successfully completed. The response is
+provided as an argument
+<2> Called in case of failure. The raised exception is provided as an argument
+
+[[java-rest-high-get-settings-response]]
+==== Get Settings Response
+
+The returned `GetSettingsResponse` allows to retrieve information about the 
+executed operation as follows:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-settings-response]
+--------------------------------------------------
+<1> We can retrieve the setting value for a particular index directly from the response as a string
+<2> We can also retrieve the Settings object for a particular index for further examination
+<3> The returned Settings object provides convenience methods for non String types
+
+If the `includeDefaults` flag was set to true in the `GetSettingsRequest`, the 
+behavior of `GetSettingsResponse` will differ somewhat.
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/IndicesClientDocumentationIT.java[get-settings-defaults-response]
+--------------------------------------------------
+<1> Individual default setting values may be retrieved directly from the `GetSettingsResponse`
+<2> We may retrieve a Settings object for an index that contains those settings with default values
```

Comment:
something is wrong in the docs. I get "Couldn't find tag get-settings-request-include-defaults" and  "get_settings.asciidoc: line 28: no callouts refer to list item 1" .